src/test/lib/DatagramTest.cpp

```diff
@@ -255,6 +257,22 @@ QuicTestDatagramSend(
                 TEST_TRUE(Client.GetDatagramsSuspectLost() > InitialLostCount);
                 CxPlatSleep(100);
 #endif
+                for (int i = 0; i < 20; i++) {
```

Comment:
Can we actually create a new test for this stuff, instead of putting it in this existing one? Also, It's possible, though unlikely, that you don't get any cancelled datagrams if they all happen to fit in the congestion control window. So, I recommend the new test queues (sends) all these datagrams on the connection **before** you call `Client.Start` to ensure you've queued up more datagrams than can be sent in the initial congestion window.